src/atomvm_esp32_flash_provider.erl

```diff
@@ -25,11 +25,11 @@
 -define(PROVIDER, esp32_flash).
 -define(DEPS, [packbeam]).
 -define(OPTS, [
-    {esptool, $e, "esptool", undefined, "Path to esptool.py"},
-    {chip, $c, "chip", undefined, "ESP chip (default auto)"},
-    {port, $p, "port", undefined, "Device port (default /dev/ttyUSB0)"},
-    {baud, $b, "baud", undefined, "Baud rate (default 115200)"},
-    {offset, $o, "offset", undefined, "Offset (default 0x210000)"}
+    {esptool, $e, "esptool", string, "Path to esptool.py"},
```

Comment:
Is the fourth argument here "string" used to validate user input?

Reply:
Not so much checking as much type coercion, I think.  I might retry making baud an integer.